test/util/blobfactory/payforblob_factory.go

```diff
@@ -23,6 +24,13 @@ import (
 
 var defaultSigner = testfactory.RandomAddress().String()
 
+const (
+	// TestMaxBlobSize is the maximum size of each blob in a blob transaction, for testing purposes
+	TestMaxBlobSize = 100
```

Comment:
[blocking]
I believe this is in bytes here

https://github.com/celestiaorg/celestia-app/blob/5787e52aed42663bf133a8649c0324b4eb45eeef/test/util/blobfactory/payforblob_factory.go#L305

and if that's the case, then should we have blobs of many different sizes?

Reply:
Can you please elaborate on your question? 
> should we have blobs of many different sizes?

Currently, all the generated Blob transactions (according to the [RandMultiBlobTxs](https://github.com/celestiaorg/celestia-app/blob/1697b1a0e2c380a72e44678dbbf83865c89fee64/test/util/blobfactory/payforblob_factory.go#L526) function), have multiple blobs of different sizes each, are you suggesting we should not do that?

Reply:
yeah sure the blobs will be of different sizes, but they will all only use a single share since the max since is denominated in bytes afaict

Reply:
I see, good point, I now updated the max blob size in https://github.com/celestiaorg/celestia-app/pull/1860/commits/a3f07b75013b290f341184d2ffc1e37dae2050aa  so that a blob can roughly span two rows, at max.

Reply:
@evan-forbes Please let me know if the changes in https://github.com/celestiaorg/celestia-app/commit/a3f07b75013b290f341184d2ffc1e37dae2050aa addresses your concern.